scripts/fast-reboot

```diff
@@ -606,34 +603,11 @@ else
     load_kernel
 fi
 
-if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
-    # Dump the ARP and FDB tables to files also as default routes for both IPv4 and IPv6
-    # into /host/fast-reboot
-    DUMP_DIR=/host/fast-reboot
-    mkdir -p $DUMP_DIR
-    FAST_REBOOT_DUMP_RC=0
-    /usr/local/bin/fast-reboot-dump.py -t $DUMP_DIR || FAST_REBOOT_DUMP_RC=$?
-    if [[ FAST_REBOOT_DUMP_RC -ne 0 ]]; then
-        error "Failed to run fast-reboot-dump.py. Exit code: $FAST_REBOOT_DUMP_RC"
-        unload_kernel
-        exit "${EXIT_FAST_REBOOT_DUMP_FAILURE}"
-    fi
-
-    FILTER_FDB_ENTRIES_RC=0
-    # Filter FDB entries using MAC addresses from ARP table
-    /usr/local/bin/filter_fdb_entries -f $DUMP_DIR/fdb.json -a $DUMP_DIR/arp.json -c $CONFIG_DB_FILE || FILTER_FDB_ENTRIES_RC=$?
```

Comment:
If this logic is being removed, I don't think we need `filter_fdb_entries` and `fast-reboot-dump` scripts any longer. What do you think?

Reply:
I am not sure if it is still relevant but it looks like those scripts are needed as well when performing config reload on dual ToR systems. https://github.com/sonic-net/sonic-utilities/pull/1465

Reply:
Good catch.